packages/local_auth/local_auth_ios/CHANGELOG.md

```diff
@@ -1,3 +1,6 @@
+## 1.0.2
+* Adds support `localizedFallbackTitle` in authenticateWithBiometrics on iOS.
```

Comment:
Nit: Please put a blank line between this and the header, to match the other entries.